src/views/v2/rsvp/details/availability.php

```diff
@@ -17,18 +17,23 @@
  * @version TBD
  */
 
-$is_unlimited = -1 === $rsvp->remaining();
+$is_unlimited     = - 1 === $rsvp->remaining();
+$is_in_stock      = $rsvp->is_in_stock();
+$has_days_to_rsvp = false;
 
+if ( $is_in_stock ) {
+	$days_to_rsvp     = $this->template( 'v2/rsvp/details/availability/days-to-rsvp', [ 'rsvp' => $rsvp ], false );
+	$has_days_to_rsvp = ! empty( $days_to_rsvp );
+}
 ?>
 <div class="tribe-tickets__rsvp-availability tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
-	<?php if ( ! $rsvp->is_in_stock() ) : ?>
+	<?php if ( ! $is_in_stock ) : ?>
 		<?php $this->template( 'v2/rsvp/details/availability/full', [ 'rsvp' => $rsvp ] ); ?>
 	<?php elseif ( $is_unlimited ) : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/unlimited', [ 'is_unlimited' => $is_unlimited ] ); ?>
+		<?php $this->template( 'v2/rsvp/details/availability/unlimited', [ 'is_unlimited' => $is_unlimited, 'has_days_to_rsvp' => $has_days_to_rsvp ] ); ?>
 	<?php else : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/remaining', [ 'rsvp' => $rsvp ] ); ?>
+		<?php $this->template( 'v2/rsvp/details/availability/remaining', [ 'rsvp' => $rsvp, 'has_days_to_rsvp' => $has_days_to_rsvp ] ); ?>
 	<?php endif; ?>
 
-	<?php $this->template( 'v2/rsvp/details/availability/days-to-rsvp', [ 'rsvp' => $rsvp ] ); ?>
-
+	<?php echo $days_to_rsvp; ?>
```

Comment:
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$days_to_rsvp'
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$days_to_rsvp'.

<sub>hash: d315b63722daa44e99c99ffda45fc6f9</sub>